src/processor.ts

```diff
@@ -128,6 +137,14 @@ const processor = new SubstrateBatchProcessor()
             // Contracts
             'Contracts.ContractEmitted',
             'Contracts.Instantiated',
+            // Spacewalk
+            'Issue.RequestIssue',
+            'Issue.ExecuteIssue',
+            'Issue.CancelIssue',
+            'Issue.IssueAmountChange',
+            'Redeem.RequestRedeem',
+            'Redeem.ExecuteRedeem',
+            'Redeem.CancelRedeem',
```

Comment:
Nit: let's also handle the other events that can modify redeem requests.

```suggestion
            'Issue.RequestIssue',
            'Issue.ExecuteIssue',
            'Issue.CancelIssue',
            'Issue.IssueAmountChange',
            'Redeem.RequestRedeem',
            'Redeem.ExecuteRedeem',
            'Redeem.CancelRedeem',
            'Redeem.LiquidationRedeem',
            'Redeem.MintTokensForReimbursedRedeem'
```

Reply:
Sure. I didn't think those could modify the initial request state, but now that we are just tracking the instance in the storage there is no effort.

Reply:
I actually added both events into the types. But I think we cannot handle `LiquidationRedeem`, since it is not part of any particular redeem (as such it doesn't return any `redeemId`), and the vaults entity we keep track just tracks the id and currency pair and not liquidation-related fields.

Reply:
Oh right, thanks for pointing it out. Since the liquidation redeem happens against the liquidation vault that is entirely managed by Spacewalk, it all happens in one go and not for a specific decentralized vault. We don't need to consider that event then, sorry for the confusion.